assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -173,9 +218,14 @@ export const baseReducer = ( state, { type, payload } ) => {
 const baseResolvers = {
 	*getUserInputSettings() {
 		const { select } = yield commonActions.getRegistry();
-		if ( ! select( STORE_NAME ).getUserInputSettings() ) {
+
+		if ( ! select( STORE_NAME ).getUserInputSettings() && select( STORE_NAME ).getUserInputState() === 'completed' ) {
```

Comment:
This seems wrong. There might be saved user input settings also when the state is not `completed`. This part shouldn't be changed, we need to only make sure that the cached user input settings take precedence over what comes from the server.

Reply:
Ah, fair point. I guess since we never store it _after_ the user input status is `complete` that's safe to do. 👍🏻